lib/pg.provider.js

```diff
@@ -200,7 +190,8 @@ class PostgresProvider extends StorageProvider {
       return;
     }
 
-    const categoryType = metaschema.categories.get(category).constructor.name;
+    const categoryType =
+      this.gs.schema.categories.get(category).constructor.name;
```

Comment:
I think it'd be better if we move every usage of `constructor.name` in the internal function/utils.
Also, the functions are called isIgnored*Category* but they accept a category type so this will also make names better represent the purpose.